fingertip.html

```diff
@@ -1,98 +1,137 @@
-        <div class="container">
-            <div class="hero">
-                <a href="https://impervious.com"><img src="images/logo.svg" class="logo" alt="Impervious" /></a>
-             
-                <h1>Fingertip</h1>
-                <div style="text-align: left; font-size: 18px;">
-                  
-                    <div style="text-align: center; margin-top: 2em;">
-                        <iframe class="video" src="https://www.youtube.com/embed/APKaRJyUnCQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-                    </div>
-                    <div style="margin: 2em 4em;">
-                        <h2>Features</h2>
-                        <ul style="margin-bottom: 4em;">
-                            <li>A lightweight decentralized Handshake resolver</li>
-                            <li>Zero trust TLS using DANE/TLSA without a centralized CA</li>
-                            <li>Access to external namespaces such as ENS</li>
-                        </ul>
-                        <div style="text-align: center; font-size: 14px;  max-width: 440px; margin: 0 auto;">
-                            <a href="https://github.com/imperviousinc/fingertip/releases/download/v0.0.1/Fingertip-v0.0.1.dmg" class="download-btn macos">Download for MacOS</a>
-                            <a href="https://github.com/imperviousinc/fingertip/releases/download/v0.0.1/Fingertip-v0.0.1.msi" class="download-btn win ">Download for Windows</a>
+        }
+    </style>
+</head>
+<body>
+<!-- Hero -->
+<div class="container">
+    <div class="hero">
+        <a href="https://impervious.com"><img src="images/logo.svg" class="logo" alt="Impervious"/></a>
+
+        <h1>Fingertip</h1>
+        <p>An <a style="text-decoration: none;" href="https://github.com/imperviousinc/fingertip">open source</a>
+            resolver for truly decentralized names.</p>
+        <div style="text-align: left; font-size: 18px;">
+            <div style="text-align: center; margin-top: 2em;">
+                <iframe
+                        class="video"
+                        src="https://www.youtube.com/embed/APKaRJyUnCQ"
+                        title="YouTube video player"
+                        frameborder="0"
+                        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                        allowfullscreen
+                ></iframe>
+            </div>
+            <div style="max-width: 650px; margin: 0 auto;">
+                <h2 style="margin-top: 2em;">Features</h2>
+                <ul style="margin-bottom: 2em; text-align: left;">
+                    <li>A lightweight decentralized Handshake resolver</li>
+                    <li>Zero trust TLS using DANE/TLSA without a centralized CA</li>
+                    <li>Access to external namespaces such as ENS</li>

```

Comment:
```suggestion
                    <li>A lightweight, decentralized Handshake resolver</li>
                    <li>Zero-trust browser security using DANE/TLSA without any centralized certificate authorities</li>
                    <li>Access to external namespaces such as <a href="https://ens.domains/">Ethereum Name Service</a></li>

```

What do you think of spelling it out for non-technical users like this?